frontend/src/components/activePage/TableInstance/RowInstanceTitle/RowInstanceTitle.tsx

```diff
@@ -27,8 +27,12 @@ const RowInstanceTitle: FC<IRowInstanceTitleProps> = ({ ...props }) => {
     gui,
   } = instance;
 
+  useEffect(() => {
+    setTitle(name);
+  }, [name]);
+
   const [edit, setEdit] = useState(false);
-  const [title, setTitle] = useState(name);
+  const [title, setTitle] = useState(prettyName ?? name);
```

Comment:
```suggestion
  const [title, setTitle] = useState(prettyName || name);
```
It could happenn that this value is empty: in that case it's better to show the name.

Reply:
ok